projects/vdk-plugins/vdk-jobs-troubleshooting/src/vdk/plugin/jobs_troubleshoot/troubleshoot_utilities/thread_dump.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import json
+import logging
+import sys
+import threading
+import traceback
+from http.server import BaseHTTPRequestHandler
+
+from vdk.internal.core.config import Configuration
+from vdk.plugin.jobs_troubleshoot.api.troubleshoot_utility import ITroubleshootUtility
+from vdk.plugin.jobs_troubleshoot.troubleshoot_configuration import (
+    TROUBLESHOOT_PORT_TO_USE,
+)
+from vdk.plugin.jobs_troubleshoot.troubleshoot_utilities.healthcheck_server import (
+    HealthCheckServer,
+)
+
+log = logging.getLogger(__name__)
+
+
+class ThreadDumpHandler(BaseHTTPRequestHandler):
```

Comment:
Provide descripton of the class purpose, usage, examples

Reply:
Done.